src/styles/main.scss

```diff
@@ -12,3 +12,24 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+
+  justify-content: center;
+  grid-template-columns: 200px 200px 200px 200px;
```

Comment:
use `repeat( number, size)` for your case it is `repeat(4, 200px)`
the same in media queries :)